infra/gcp/terraform/k8s-infra-prow-build/main.tf

```diff
@@ -105,6 +105,27 @@ module "prow_build_nodepool_n1_highmem_8_localssd" {
   service_account           = module.prow_build_cluster.cluster_node_sa.email
 }
 
+module "prow_build_nodepool_c4_highmem_8_localssd" {
+  source       = "../modules/gke-nodepool"
+  project_name = module.project.project_id
+  cluster_name = module.prow_build_cluster.cluster.name
+  location     = module.prow_build_cluster.cluster.location
+  node_locations = [
+    "us-central1-b",
+    "us-central1-c",
+    "us-central1-f",
+  ]
+  name            = "pool6"
+  initial_count   = 1
+  min_count       = 1
+  max_count       = 80
+  machine_type    = "c4-highmem-8"
+  disk_size_gb    = 100
+  disk_type       = "hyperdisk-balanced"
+  service_account = module.prow_build_cluster.cluster_node_sa.email
+  taints          = [{ key = "dedicated", value = "sig-testing", effect = "NO_SCHEDULE" }]
```

Comment:
I want to move all the sig-testing prowjobs on it.

Reply:
Can we bump the disk to 500gb?

Reply:
Yes. But why ? 
we currently have 100 GB and there's no report about disk usage or IOPS ?

Reply:
> I want to move all the sig-testing prowjobs on it.

We probably should not move PR/release-blocking jobs for Kubernetes and KIND (because the presubmits there need to check safety for k/k) until after code thaw settles.

We can move test-infra, boskos, prow etc though.

Reply:
> we currently have 100 GB and there's no report about disk usage or IOPS ?

Users don't have good visibility into IOPS.

I'm not sure how big we should aim for though.

Reply:
Also don't we have local SSD currently?

Reply:
>   ephemeral_local_ssd_count = 2 # each is 375GB

Reply:
C4 don't support Local SSD. the `Hyperdisk` disks should be enough in theory.

Reply:
Having bigger disks should compensate the need of local SSD. we should probably try also with `hyperdisk-extreme` disks.

Reply:
> We probably should not move PR/release-blocking jobs for Kubernetes and KIND (because the presubmits there need to check safety for k/k) until after code thaw settles.

Target the periodics (only master-informing) for k/k seems like a good start before code thaw ? we probably want to observe disk performance during peak traffic.

Reply:
Bumped to 500 GB for the moment. We can revisit this later.